test/CTestList.cmake

```diff
@@ -37,19 +37,21 @@ function(add_test_r TEST_NAME)
     if(AMR_WIND_ENABLE_CUDA)
       set(FCOMPARE_TOLERANCE "-r 1e-10 --abs_tol 1.0e-12")
     endif()
-    # Use fcompare to test diffs in plots against gold files
-    if(AMR_WIND_TEST_WITH_FCOMPARE)
-      set(FCOMPARE_COMMAND "&& ${FCOMPARE_EXE} ${FCOMPARE_TOLERANCE} ${PLOT_GOLD} ${PLOT_TEST}")
-    endif()
     if(AMR_WIND_ENABLE_MPI)
       set(NP 4)
       set(MPI_COMMANDS "${MPIEXEC_EXECUTABLE} ${MPIEXEC_NUMPROC_FLAG} ${NP} ${MPIEXEC_PREFLAGS}")
+      set(FCOMPARE_COMMAND "${MPIEXEC_EXECUTABLE} ${MPIEXEC_NUMPROC_FLAG} 1 ${MPIEXEC_PREFLAGS} ${FCOMPARE_EXE} ${FCOMPARE_TOLERANCE} ${PLOT_GOLD} ${PLOT_TEST}")
```

Comment:
I don't understand `NP=1` here. Why use `mpirun`?

Reply:
Actually, maybe the question here is why does it segfault without an `mpirun`? I thought if `np=1`, then `mpirun` shouldn't be necessary.

Reply:
You can take this up with NREL Eagle. :)

Reply:
Actually, I'm thinking we just let `fcompare` run with `NP` and use the already existing `MPI_COMMANDS`?

Reply:
That's fine with me too.